### DISCUSSION TITLE
Is it possible to use Lowdefy to generate a website to run somewhere else?

### DISCUSSION THREAD

**Discussion by IvanDrag0** [lowdefy/lowdefy#1659] [Questions and Help] [answered]
I really like how easy it is to generate websites using the Lowdefy YAML schema/process. My question is: if I just want to create a webpage that I want to run outside of Lowdefy, is it possible to use it to generate the page itself (HTML, CSS, etc.) and save it somewhere? I don't need any actions or anything similar. I would just like a quick way to generate high level website pages/templates.

**Comment by Gervwyk** [comment, marked as answer]
HI @IvanDrag0 
For v4, Lowdefy runs as a Next.js app. So you can deploy and run it anywhere your run Next.js. Check out the Lowdefy website, which is written in Lowdefy, and hosted on vercel: https://github.com/lowdefy/lowdefy/tree/main/packages/website
